internal/output/output_result.go

```diff
@@ -372,7 +372,7 @@ func processSource(packageSource models.PackageSource) map[string]SourceResult {
 	// If no packages with issues are found, mark the ecosystem as empty.
 	if len(packageSource.Packages) == 0 {
 		sourceResults[""] = SourceResult{
-			Name:      packageSource.Source.String(),
```

Comment:
I think this is used by HTML output. If you want to remove the type, you might need to also modify https://github.com/google/osv-scanner/blob/dbac12ea035c7fcf8badb958e915ca38246bb8d9/internal/output/html/package_view_template.gohtml#L7

I don't think it's bad to have type printing out though. This is helpful for container scanning.

Reply:
I think we don't include the type in the table output mainly because of the width of the table. But we don't have this concern for vertical or HTML outputs, so I think we can try to provide more details there.

Reply:
Two issues with that:
  1. this function is currently assuming all formats want to display that when that's not the case, so the table format then needs to undo this with a chain of ugly `TrimPrefix` calls which feels more brittle than having the other formats handle this themselves (as they already have access to both the `Name` and the `Type`)
  2. our output fixtures are not entirely representative of the real world as I believe we always provide the absolute path which can be a lot longer, so even for the vertical output there's a good chance this can be get pushed over 80 chars so including the type has a not-zero cost (see #2034 for more, though it still doesn't entirely show this off because we replace most of the path with `<rootdir>` as part of our snapshot normalizations)